fetch.bs

```diff
@@ -760,7 +764,12 @@ fetch("https://victim.example/naïve-endpoint", {
     </div>
 
    <dt>Otherwise
-   <dd><p>Return false.
+   <dd>
+    <p>If <var>name</var> does not begin with the string "<code>sec-</code>", return false.
+
+    <p class=note>As all headers beginning with "<code>Sec-</code>" are <a>forbidden header
```

Comment:
```suggestion
    <p class=note>As all headers starting with `<code>Sec-</code>` are <a>forbidden header
```

Reply:
Thank you. :) I was searching for variants of "begin" and "prefix" without luck!